icu4j/main/classes/core/src/com/ibm/icu/text/DateIntervalFormat.java

```diff
@@ -377,7 +377,8 @@ private SpanField(String name) {
         /**
          * serialization method resolve instances to the constant
          * DateIntervalFormat.SpanField values
-         * @draft ICU 64
+         * @internal
+         * @deprecated This API is ICU internal only.
          */
         @Override
```

Comment:
As I mentioned my previous comment, you need @Deprecated annotation here.

--------
        /**
         * serialization method resolve instances to the constant
         * DateIntervalFormat.SpanField values
         * @draft ICU 64
         * @internal
         * @deprecated This API is ICU internal only.
         */
        @Deprecated
        @Override
--------

See `@Deprecated` out of API comment, next to `@Override`.  It looks this is not explained in the same BRS task section - I'll update the description.

Reply:
Thanks for the additional explanation. I've updated these 3 files.